tests/integration_tests/test_graph_execution.py

```diff
@@ -69,33 +68,17 @@ def test_experiment_multiple_seq_seq(csv_filename, output_features):
     run_experiment(input_features, output_features, dataset=rel_path)
 
 
-@pytest.mark.skip(reason="Issue #1333: Sequence output features.")
 @pytest.mark.distributed
-@pytest.mark.parametrize("dec_beam_width", [3])
-@pytest.mark.parametrize("dec_attention", ["bahdanau"])
-@pytest.mark.parametrize("dec_cell_type", ["lstm"])
-@pytest.mark.parametrize("enc_cell_type", ["lstm"])
+@pytest.mark.parametrize("dec_cell_type", ["rnn"])
+@pytest.mark.parametrize("enc_cell_type", ["rnn"])
 @pytest.mark.parametrize("enc_encoder", ["embed"])
-def test_sequence_generator(enc_encoder, enc_cell_type, dec_cell_type, dec_attention, dec_beam_width, csv_filename):
```

Comment:
maybe add a todo here to remember to add attention and beam search back?

Reply:
I've gone ahead and removed this particular test completely as it's highly overlapping the test we already have in run_experiment.

I'm planning to test all of these aspects of decoder within test_sequence_generator unit tests.